### DISCUSSION TITLE
How to run MatchZy

### DISCUSSION THREAD

**Discussion by SaltSenbei** [kus/cs2-modded-server#56] [Q&A]
Hello, I would like to use MatchZy but I don't know how to call it (!pause, !swap, etc.).

**Comment by Danijongo** [comment]
You can refer to MatchZy's own FQA for that here: https://shobhit-pathak.github.io/MatchZy/commands/

**Comment by kus** [comment]
@SaltSenbei assuming you have switched to `Competitive` using `!modes` or `exec comp` (from server console) you simply type them in chat, they are prefixed with `.` as @Danijongo pointed out.

https://shobhit-pathak.github.io/MatchZy/commands/
